doc/jsk_topic_tools/class/python_diagnostic_transport.md

```diff
@@ -0,0 +1,28 @@
+# DiagnosticTransport (Python)
+
+## Description
+
+This class is a subclass of [`ConnectionBasedTransport`](python_connection_based_transport.html).
+Instances of this class can publish diagnostics messages.
+This is an abstruct class.
+
```

Comment:
If this feature is designed for human operator to observe the health of robot system on  `rqt_robot_monitor`?, please add image of diagnostic viewer.

Reply:
OK. I'll add it.

Reply:
Added it.
https://github.com/jsk-ros-pkg/jsk_common/blob/ca3348bec3dd638f9cebc24bbcea3b79ef576dee/doc/jsk_topic_tools/class/python_connection_based_transport.md